### ISSUE TITLE
sorting

### ISSUE THREAD

**Issue by arist0v** [createcandle/candleappstore#1]
It would be nice if the app could be sort by : 
- candle app vs webthings apps(or all)
- adapter vs extension vs notifier
it would help sort and discover apps

**Comment by createcandle** [comment]
The first one would actually be harder to implement, as the data currently doesn't distinguish between the two systems.
- What would be value of this be to a user who only every uses Candle? The target audience will likely never use the Webthings Gateway.

Adapter vs extension vs notifier data is available to the app store already, so would be easier to add. I just didn't see the value for two reasons:
- It only seems a useful distinction if you know a bit more about how the technology works? Do normal users understand the difference?
- Practically, most Candle addons are both adapters and extensions at the same time, muddying the distinction even more.

In the end I worry that it might be confusing to some people.

**Comment by arist0v** [comment]
`The first one would actually be harder to implement, as the data currently doesn't distinguish between the two systems.

What would be value of this be to a user who only every uses Candle? The target audience will likely never use the Webthings Gateway.`

the value could be for developper but yeah it may be just for me can forget this one

`Adapter vs extension vs notifier data is available to the app store already, so would be easier to add. I just didn't see the value for two reasons:

It only seems a useful distinction if you know a bit more about how the technology works? Do normal users understand the difference?
Practically, most Candle addons are both adapters and extensions at the same time, muddying the distinction even more.
In the end I worry that it might be confusing to some people.`

for advanced user it can be a must, to help easily sort stuff(and i often user adapter to discover new brand of devices)

an addon that provide an adapter is adapter even if it use extension to provide adapter configuration, the type in the manifest should be the valide one

**Comment by createcandle** [comment]
> an addon that provide an adapter is adapter even if it use extension to provide adapter configuration, the type in the manifest should be the valide one

That's true. Still, I'm worries it just adds something new that new users will feel they have to learn/understand.

The app store has a tagging system and one of the tags has sort of the same function.

**Comment by arist0v** [comment]
wich tags?

**Comment by createcandle** [comment]
When you click on an addon to see its details you will see some tags in the rop-right corner

**Comment by arist0v** [comment]
yeah but you still can't browse only adapter addon or extension addon or
notifier addon

and the use for a end user could be:

what kind of notification is available?

browse notifier addons

**Comment by createcandle** [comment]
The next version of the Candle Apl Store will have extra tags indicating extension type if developer mode is active.
